ql/src/test/queries/clientpositive/serde_opencsv.q

```diff
@@ -28,9 +29,10 @@ ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.OpenCSVSerde'
 WITH SERDEPROPERTIES(
   "separatorChar" = ",",
   "quoteChar"     = "\'",
-  "escapeChar"    = "\\"
+  "escapeChar"    = "\\",
+  "serialization.encoding" = "ISO8859_1"
 ) stored as textfile;
 
 LOAD DATA LOCAL INPATH "../../data/files/opencsv-data.txt" INTO TABLE serde_opencsv;
 
-SELECT count(*) FROM serde_opencsv;
+SELECT * FROM serde_opencsv;
```

Comment:
was there any purpose of the original count(*) query? without knowing the history, I believe we can leave both queries here

Reply:
thank you for the review. will update.